package Exception;

public class OneTryMultipleCatch 
{

	public static void main(String[] args) 
	{
		int a[] = new int[4];
		int i,j,k=0;
		i = 8;
		j = 2;
		
		try
		{
			
			k = i/j;
			for(int c=0;i<=4;i++)
			{
				a[c] = c+1;
			}
			
			for(int value : a)
			{
				System.out.println(value);
			}
		}
		catch(ArithmeticException e)
		{
//			System.out.println(e);
			System.out.println("Cannot devide by Zero "+ e);
		}
		catch(ArrayIndexOutOfBoundsException e)
		{
			System.out.println(e);
		}
		System.out.println(k);
	}

}
